benchmarks/bench_serve.py

```
@@ -0,0 +1,338 @@
+    print(f"  URL:         {args.url}")
+    print(f"  Model:       {args.model}")
+    print(f"  Max tokens:  {args.max_tokens}")
+    print(f"  Temperature: {args.temperature}")
+    print(f"  Prompts:     {len(PROMPTS)}")
+    print()
+
+    # Check server is alive
+    try:
+        r = requests.get(f"{args.url}/health", timeout=5)
+        print(f"Server health: {r.status_code}")
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)